packages/webapp/src/containers/Animals/Inventory/index.tsx

```diff
@@ -231,6 +240,12 @@ function AnimalInventory({ isCompactSideMenu }: AnimalInventoryProps) {
               }}
             />
           ) : null}
+          <RemoveAnimalsModal
+            isOpen={removalModalOpen}
+            onClose={() => setRemovalModalOpen(false)}
+            onConfirm={handleAnimalOrBatchRemoval}
+            showSuccessMessage={false}
+          />
```

Comment:
The success message text mentions historical inventory, so I don't think it needs to be displayed until that has been built.